src/views/parents/l10n.json

```diff
@@ -0,0 +1,26 @@
+    "parents.onlineCommunity": "What is the Scratch online community?",
+    "parents.onlineCommunityDescription": "When participating in the Scratch online community, members can explore and experiment in an open learning community with other Scratch members from all backgrounds, ages, and interests. Members can share their work, get feedback, and learn from each other.",
+    "parents.guidelines": "What are the guidelines for the Scratch online community?",
+    "parents.guidelinesDescription": "The MIT Scratch Team works with the community to maintain a friendly and respectful environment for people of all ages, races, ethnicities, religions, sexual orientations, and gender identities. You can help your child learn how to participate by reviewing the <a href=\"/guidelines\">Community Guidelines</a> together. Members are asked to comment constructively and to help keep the website friendly by reporting any content that does not follow the community guidelines. The Scratch Team works each day to manage activity on the site and respond to reports, with the help of tools such as the <a href=\"https://www.inversoft.com/products/profanity-filter\">CleanSpeak</a> profanity filter.",
+    "parents.privacyPolicy": "What is your privacy policy?",
+    "parents.privacyPolicyDescription": "To protect children's online privacy, we limit what we collect during the signup process, and what we make public on the website. We don't sell or rent account information to anyone. You can find out more about our privacy policy on our <a href=\"/privacypolicy\">Privacy Policy</a> page.",
+    "parents.noOnline": "Is there a way to use Scratch without participating online?",
+    "parents.noOnlineDescription": "Yes, the Scratch offline editor lets you create projects without joining or accessing the online community. Visit the <a href=\"/scratch2download/\">Scratch 2.0 offline editor</a> download page for instructions on how to install it on your computer. (If your computer does not support the latest version, try the <a href=\"/scratch_1.4/\">Scratch 1.4 offline editor</a>.)",
+    "parents.parentsSaying": "Learn More About Scratch",
+    "parents.parentsSayingDescription": "We often receive emails from parents thanking us for Scratch. Here are some examples:",
+    "parents.Questions": "Questions?",
+    "parents.QuestionsDescription": "To find out more about Scratch, please see <a href=\"/faq/\"Frequently Asked Questions</a>. You can also ask questions in the <a href=\"/discuss/\"discussion forums</a>. If you need to contact our staff team directly, click <a href=\"/contact-us/\"Contact Us</a> at the bottom of any page."
```

Comment:
I think there are a few missing `>` closing tags here. What I'm seeing locally is:
<img width="773" alt="screen shot 2016-10-14 at 9 36 56 am" src="https://cloud.githubusercontent.com/assets/1815772/19389411/ed971540-91f1-11e6-86ae-868a459b4344.png">